src/app/content/reducer.ts

```diff
@@ -17,6 +17,7 @@ import { getPageSlug } from './utils/archiveTreeUtils';
 export const initialState = {
   highlights: initialHighlightState,
   loading: {},
+  myHighlightsOpen: initialHighlightState.myHighlightsOpen,
```

Comment:
this is already dealt with in `src/app/content/highlights/reducer.ts`, you don't need it here